extensions/2.0/Vendor/EXT_manifold/schema/glTF.EXT_manifold.schema.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Please rename this file to `mesh.EXT_mesh_manifold.schema.json`.  (So, "mesh" instead of "glTF" at the front, and add "mesh" again to the extension name).